app/Decsys/Models/Webhooks/EventTriggerFilters.cs

```diff
@@ -0,0 +1,10 @@
+using Decsys.Constants;
+using Newtonsoft.Json;
+
+namespace Decsys.Models.Webhooks;
+
+public class EventTriggerFilters
+{
+    [JsonProperty(WebhookEventTypes.PAGE_NAVIGATION)]
+    public List<PageNavigationFilters> PageNavigation = new();
```

Comment:
Can we make this List nullable and default to null?
```suggestion
    public List<PageNavigationFilters>? PageNavigation {get; set;}
```